libs/dh/metering-point/shared/ui-util/src/lib/dh-yes-no.pipe.ts

```diff
@@ -14,20 +14,23 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { NgModule, Pipe, PipeTransform } from '@angular/core';
-import { TranslocoModule, TranslocoService } from '@ngneat/transloco';
+import { Pipe, PipeTransform, inject } from '@angular/core';
+import { TranslocoService } from '@ngneat/transloco';
 
 export type TValue = string | boolean | undefined | null;
 export const pipeName = 'dhYesNo';
 
 @Pipe({
   name: pipeName,
   pure: false,
+  standalone: true,
 })
 export class DhYesNoPipe implements PipeTransform {
+  private transloco: TranslocoService = inject(TranslocoService);
```

Comment:
nitpick: the `inject` function is type-safe, so TS should be able to infer the type automatically.

```suggestion
  private transloco = inject(TranslocoService);
```